automation_tools/__init__.py

```diff
@@ -1763,22 +1774,25 @@ def create_openstack_instance(instance_name, image_name, flavor_name, ssh_key):
         network=network.id
     )
     # Assigning floating ip to instance
+    timeup = time.time()+(int(timeout)*60)
     while True:
         try:
+            if time.time() > timeup:
+                print('The timeout for assigning the floating IP has reached!')
+                sys.exit(1)
             instance.add_floating_ip(floating_ip)
+            print('SUCCESS!! The flaoting IP has been assigned to instance!')
```

Comment:
Would it be helpful to note information about `floating_ip` or `instance` in these messages?